lib/workos/user_management.rb

```diff
@@ -275,6 +275,33 @@ def verify_email(magic_auth_challenge_id:, code:)
 
         WorkOS::User.new(response.body)
       end
+
+      # Updates user user password.
+      #
+      # @param [String] id The unique ID of the User.
+      # @param [String] new_password The new password to set for the user.
+      #
+      # @return WorkOS::User
+
+      sig do
+        params(
+          id: String,
+          password: String,
+        ).returns(WorkOS::User)
+      end
+      def update_user_password(id:, password:)
+        response = execute_request(
+          request: put_request(
+            path: "/users/#{id}",
```

Comment:
In the Node SDK the URL is "/users/#{id}/password"

🤔

Reply:
fixed 👍